forms/offer/products/OfferBuildingMaterials.tsx

```diff
@@ -51,7 +53,11 @@ export const OfferBuildingMaterials: FC<IProps> = ({ counties, onSubmit, categor
       quantity: yup.number().min(1, t('error.quantity.minOne')).typeError(t('error.must.be.number')),
       unit_type: yup.string().required(t('error.unitType.required')),
       packaging_type: yup.string().required(t('error.packagkingType.required')),
-      expiration_date: yup.mixed().typeError(t('error.must.be.date')),
+      expiration_date: yup
+        .string()
+        .required(t('validation.required'))
+        .test(validDateTestName, t('validation.date.invalid'), validDateTest)
```

Comment:
`validDateTest` ?

Reply:
naming is hard 
is `validDateTestFunc` better ?

Reply:
if it returns a boolean, I would go with `isDateValid`, not sure what it returns though :)